### ISSUE TITLE
regression in server mode 0.9.13a0 compared to 0.9.12

### ISSUE THREAD

**Issue by marcbernot** [holoviz/holoviz#116]
By trying to solve #101 I noticed a bug with the taxi example in 13_Deploying_Bokeh_Apps.ipynb in server mode.
The app is launched with `bokeh serve --show 13_Deploying_Bokeh_Apps.ipynb` 
With PyViz 0.9.12, it is working just fine and you can choose pickup then dropoff again and everything updates.
With PyViz 0.9.13a0 you can choose pickup and it updates. But it does not update anymore if you choose dropoff again.

Here is a small part of the error message ; panel and/or param are involved.
```
  File "/anaconda3/envs/pyviz0913/lib/python3.6/site-packages/param/parameterized.py", line 1055, in _call_watcher
    watcher.fn(self_._update_event_type(watcher, event, self_.self_or_cls.param._TRIGGER))
  File "/anaconda3/envs/pyviz0913/lib/python3.6/site-packages/panel/param.py", line 386, in update_pane
    index = parent.children.index(old_model)
ValueError: Figure(id='8ae97421-026b-47d9-ba14-337452b14043', ...) is not in list
```

The problem does not occur within the notebook.

**Comment by philippjfr** [comment]
Should be fixed by the panel PR referenced above, we'll release a new pyviz release shortly.

**Comment by marcbernot** [comment]
Thanks @philippjfr , it is indeed fixed by PyViz 0.9.13a2 that uses panel 0.1.2.
I close the issue.
